### ISSUE TITLE
Speech: allow `auto` setting for language to derive from display language

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#206321]
Fyi @TylerLeonhardt

**Comment by bpasero** [comment]
**Verification:**
* install the speech extension: https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-speech
* make sure `accessibility.voice.speechLanguage` is configured to `auto` (it should be by default)
* switch VS Code display language to another language such as german or portuguese
* start voice dictation in a text editor via "Start Dictation in Editor"
* verify you get prompted to install the related language pack for the display language you picked
